### PR TITLE
chore: manually download python 2.7 in generation workflow

### LINKED ISSUE #17547: None of the clients is released since Jun 18
The last time an Apiary Client library was released is [Jun 18](https://github.com/googleapis/google-api-java-client-services/pull/17537), most of the [Codegen](https://github.com/googleapis/google-api-java-client-services/actions/workflows/codegen.yaml) job has been failing for a while, we need to investigate and fix the issue to resume the release of Apiary client libraries.

### PR DESCRIPTION
**Pull request by diegomarquezp**
Fixes #17547

### CODE REVIEW

File: .github/workflows/generate.yaml [hunk 1]
```diff
@@ -30,9 +30,12 @@ jobs:
...
           fetch-depth: 1
           path: discovery-artifact-manager
-      - uses: actions/setup-python@v4
-        with:
-          python-version: 2.7.18
+      - run: |
+          sudo apt update
+          sudo apt install python2
+          python2 --version
+          curl https://bootstrap.pypa.io/pip/2.7/get-pip.py -o get-pip.py
```
Comment (diegomarquezp) [inline .github/workflows/generate.yaml:37]: I tried `apt-get install python-pip` but it was not available (may need a source to be manually added).
`ensure-pip` was "disabled in this version of Ubuntu"

File: .github/workflows/generate.yaml [hunk 2]
```diff
@@ -30,9 +30,12 @@ jobs:
           repository: googleapis/discovery-artifact-manager
           fetch-depth: 1
           path: discovery-artifact-manager
-      - uses: actions/setup-python@v4
-        with:
-          python-version: 2.7.18
+      - run: |
+          sudo apt update
+          sudo apt install python2
+          python2 --version
```
Comment (blakeli0) [inline .github/workflows/generate.yaml:36]: Can we echo something like "Using Python version 2.x"?

**Review by blakeli0 — COMMENTED** [review]
LGTM if the current generation scripts does not work with Python3

**Comment by diegomarquezp** [comment]
I tried python 3 (#17550) but I'm getting syntax errors. We may want to refactor it if we want to migrate to 3.x. @blakeli0 Maybe we can leave it as a stretch for Q3?

File: .github/workflows/generate.yaml (hunk 2 above)
Comment (diegomarquezp) [reply .github/workflows/generate.yaml]: replaced with `echo` message

**Comment by blakeli0** [comment]
> I tried python 3 (#17550) but I'm getting syntax errors. We may want to refactor it if we want to migrate to 3.x. @blakeli0 Maybe we can leave it as a stretch for Q3?

Yes, let's keep it out of the scope of this PR.

**Review by blakeli0 — APPROVED** [review]

**Pull request merged**